### PR TITLE
Allow passing unlimited args to ^entry

### LINKED ISSUE #82: Multi word queries for ^entry are broken
god damn

### PR DESCRIPTION
**Pull request by IreTheKID**
Closes #82 (I hope) by catching all arguments and joining them into `search_query`.

### CODE REVIEW

**Comment by Codemonkey51** [comment]
Read my comments on #82

**Comment by Codemonkey51** [comment]
Read my comments on #82

**Comment by jetstream0** [comment]
@Codemonkey51 I looked at the [docs](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html?highlight=argument) and what @IreTheKID did seems good
image

**Comment by Codemonkey51** [comment]
> @Codemonkey51 I looked at the [docs](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html?highlight=argument) and what @IreTheKID did seems good
> image

Do you understand that mat made his own lib for commands lmao, it doesn't use normal discord.ext

File: repldex/discordbot/commands.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ async def link_source(message):
 
 
 @betterbot.command(name='entry', allowed=True)
-async def show_entry(message, search_query: str):
+async def show_entry(message, *, search_query: str):
```
Comment (Codemonkey51) [inline repldex/discordbot/commands.py:122]: I'm pretty sure that won't work with repldex, this should be locally by an admeen

cc: @mat-1

**Review by Codemonkey51 — CHANGES_REQUESTED** [review]

File: repldex/discordbot/commands.py (hunk 1 above)
Comment (Codemonkey51) [reply repldex/discordbot/commands.py:122]: Tested*

Comment (Codemonkey51) [reply repldex/discordbot/commands.py:122]: The reason it may not work is bc repldex doesn't use normal discord.ext but a custom thing made by mat

**Comment by IreTheKID** [comment]
lmao I guess I should just close this then

File: repldex/discordbot/commands.py (hunk 1 above)
Comment (IreTheKID) [reply repldex/discordbot/commands.py:122]: I couldn't get a local bot working for the life of me, so ig an admin should test this.

**Comment by Codemonkey51** [comment]
> lmao, I guess I should just close this then, I only know how to use `discord.py`

Lol its modded discord.py

**Review by jetstream0 — COMMENTED** [review]
Oh... then I guess we need to figure out what the refactor thing broke then

File: repldex/discordbot/commands.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (IreTheKID) [reply repldex/discordbot/commands.py:122]: OK, that works. I'm not too sure if mat's changes to the `discord.py` lib will has any effect on arg parsing or not. If not, then that should fix the issue.

[the remark this answers is not in the document]
Comment (Codemonkey51) [reply repldex/discordbot/commands.py:122]: @IreTheKid @jetstream0 mat's lib completely overhauls arg parsing

**Pull request closed without merging**